src/Main.java

```diff
@@ -1,5 +1,122 @@
+import java.sql.SQLOutput;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        task1();
+        task2();
+        task3();
+        task4();
+        task5();
+        task6();
+        task7();
+
+    }
+
+    public static void task1() {
+        System.out.println("Задача1");
+        int age = 18;
+        if (age >= 18) {
+            System.out.println("Если возраст человека равен " + age + " то он совершеннолетний");
+        } else {
+            System.out.println("Если возраст человека не равен " + age + " он не достиг совершеннолетия, нужно немного подождать");
+        }
+    }
+
+    public static void task2() {
+        System.out.println("Задача2");
+        int temperature = 5;
+        if (temperature < 5) {
+            System.out.println("На улице ниже 5 градусов,нужно надеть шапку");
+        }
+        if (temperature >= 5) {
+            System.out.println("На улице  5 градусов,можно идти без шапки");
+        }
+
+    }
+
+    public static void task3() {
+        System.out.println("Задание3");
+        int speed = 60;
+        int maxspeed = 65;
+        if (speed > maxspeed) {
+            System.out.println("Если скорость больше " + speed + " , то придется заплатить штраф ");
+        }
+        if (speed <= maxspeed) {
+            System.out.println("Если скорость " + speed + " , то   можно ездить спокойно");
+        }
+    }
+
+    public static void task4() {
+        System.out.println("Задание4");
+        int age = 2;
+        int age1 = 6;
+        int age2 = 7;
+        int age3 = 18;
+        int age4 = 24;
+        int age5 = 60;
+        int x = 11;
+        if (age <= x) {
+            System.out.println("если человеку меньше " + age + " лет, то ему пора спать");
+        }
+        if (age == 2 && age1 == 6) {
+            System.out.println("Если человеку от 2 до 6 лет, то ему нужно ходить в детский сад.");
+        }
+        if (age2 >= 7 && age3 == 18) {
+            System.out.println("Если человеку от 7 до 18 лет, то ему нужно ходить в школу.");
+        }
+        if (age3 >= 19 && x <age4) {
+            System.out.println("Если человеку больше 18 лет, но меньше 24, то его место в университете");
+        }
+        if (age4 > x) {
+            System.out.println("А если человеку больше 24, то ему пора ходить на работу.");
+        }
+        if (age5 >= x) {
+            System.out.println("А если человеку больше 60, то он может отдохнуть.");
+        }
```

Comment:
Немного путаешь знаки меньше и больше:
```suggestion
        if (x < age) {
            System.out.println("если человеку меньше " + age + " лет, то ему пора спать");
        }
        if (x >= age && x =< age1) {
            System.out.println("Если человеку от 2 до 6 лет, то ему нужно ходить в детский сад.");
        }
        if (x >= age2 && x =< age3) {
            System.out.println("Если человеку от 7 до 18 лет, то ему нужно ходить в школу.");
        }
        if ((x > age3 && x < age4) {
            System.out.println("Если человеку больше 18 лет, но меньше 24, то его место в университете");
        }
        if (x > age4) {
            System.out.println("А если человеку больше 24, то ему пора ходить на работу.");
        }
        if (x > age5) {
            System.out.println("А если человеку больше 60, то он может отдохнуть.");
        }
```